DependencyInjection/HWIOAuthExtension.php

```diff
@@ -52,7 +53,15 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         // set current firewall
-        $container->setParameter('hwi_oauth.firewall_name', $config['firewall_name']);
+        if (empty($config['firewall_names']) && !isset($config['firewall_name'])) {
+            throw new InvalidConfigurationException('The child node "firewall_name" or "firewall_names" at path "hwi_oauth" must be configured.');
+        } elseif (!empty($config['firewall_names']) && isset($config['firewall_name'])) {
+            $config['firewall_names'] = array_merge(array($config['firewall_name'], $config['firewall_names']));
+        } elseif (empty($config['firewall_names']) && isset($config['firewall_name'])) {
+            @trigger_error('The child node "firewall_name" at path "hwi_oauth" is deprecated since version 0.3.9 and will be removed in version 0.4.0. Use "firewall_names" instead.', E_USER_DEPRECATED);
```

Comment:
> In next version, so it should be `0.5`

Could be fine to release a stable version (`1.0`) soon and follow semantic versionning.